CHANGES.md

```diff
@@ -5,6 +5,10 @@
 ##### Breaking Changes :mega:
 
 - Tilesets and entities now use `ModelExperimental` by default. This can still be changed by setting `ExperimentalFeatures.enableModelExperimental = false`. [#10530](https://github.com/CesiumGS/cesium/pull/10530)
+- Built `Cesium.js` is no longer AMD format. This may or may not be a breaking change depending on how you use Cesium in your app. See our [blog post](TODO) for the full details. [#10399](https://github.com/CesiumGS/cesium/pull/10399)
+  - If you were ingesting individual ESM-style modules from the combined file `Build/Cesium/Cesium.js` or `Build/CesiumUnminified/Cesium.js`, instead use `Build/Cesium/index.js` or `Build/CesiumUnminified/index.js` respectively.
```

Comment:
Doesn't Cesium now 100% require a bundler to use modules? (And a bundler that allows for node resolution rules at that). Or are we still embedding third party code?

Reply:
We're embedding the third-party code in these builds. In the blog post I mention that production apps that use their own bundler should be using modules directly `Source`, which does not have third party code bundled. Which is what we've previously recommended as well if I understand correctly.

Reply:
We have previously recommended it, but since we embedded ThirdParty code in source, you didn't _need_ a bundler to use them. Now you do.

Another problem I just noticed is that `package.json` lists _everything_ as a dev dependency, which may not be correct because that means if I `npm install Cesium` and then try to use `Source` it's going to pull in modules that don't exist because they are not a `dependency` or `peerDependency`.

Reply:
Gotcha, I see the distinction. For the point about the bundler, I will update both `CHANGES.md` and the blog post with the requirement.

For the latter, I will `package.json` such that modules in `Source/ThirdParty` are now marked as direct dependencies. Sound correct?

Reply:
Hey all, speaking of `package.json`, would this be a good time to address https://github.com/CesiumGS/cesium/issues/9212 as well?  It's really important to get the exports right, I had a workaround but had to get rid of it to be able to use [another important dependency](https://github.com/angular/components/issues/25305) at all...

Reply:
Hi @thw0rted, thanks for pointing this out! We have some additional thoughts on related improvements beyond just this PR, including considering https://github.com/CesiumGS/cesium/issues/9212. We'll add our thoughts or any updates to that issue.